lxd/daemon_config.go

```diff
@@ -210,7 +210,8 @@ func daemonConfigInit(db *sql.DB) error {
 	for k, v := range dbValues {
 		_, ok := daemonConfig[k]
 		if !ok {
-			logger.Error("Found invalid configuration key in database", log.Ctx{"key": k})
+			logger.Warn("Found unknown configuration key in database", log.Ctx{"key": k})
```

Comment:
I thought about it, I'm fine with keeping Error, but that gives the impression that LXD functionality is impacted by that, whereas it will just ignore the key, that's why I went with Warn